packages/api/src/graphql/blocks.ts

```diff
@@ -442,8 +443,20 @@ export const GraphQLLinkPageBreakBlock = new GraphQLObjectType<LinkPageBreakBloc
   name: 'LinkPageBreakBlock',
   fields: {
     text: {type: GraphQLString},
+    richText: {type: GraphQLNonNull(GraphQLRichText)},
     linkURL: {type: GraphQLString},
-    linkText: {type: GraphQLString}
+    linkText: {type: GraphQLString},
+    linkTarget: {type: GraphQLString},
+    hideButton: {type: GraphQLBoolean},
```

Comment:
i would make this also `GraphQLNonNull`

Reply:
why? its a boolean

Reply:
Yes. That's why. A boolean should never be null.

Reply:
does it harm or break the functionality?

Reply:
(afik i see in current setup there is no migration needed, for nonNull i think it is)

Reply:
What are you talking about? I'd like you to updat ethe GraphQL schema to make the boolean a NonNull since a boolean is always true or false.